parsec/core/gui/file_table.py

```diff
@@ -103,6 +104,7 @@ def __init__(self, *args, **kwargs):
         self.previous_selection = []
         self.setColumnCount(len(Column))
         self.config = None
+        self.has_timestamped_file = False
```

Comment:
Don't like this variable name. It's not that the workspace has a timestamped file, they all do, it's that the workspace itself is mounted at a certain timestamp. Something like `is_timestamped_workspace` would be more appropriate.